torch/library.py

```diff
@@ -1,16 +1,20 @@
+from matplotlib.style import library
```

Comment:
what's going on with this

Reply:
I ran the linter and ...

Reply:
it's a good idea to sanity check linter output LOL

Reply:
I didn't realize linter could make such crazy modifications lol. but yeah will keep this in mind in future